app/presenters/report_presenter.rb

```diff
@@ -16,7 +16,19 @@ def deadline
     I18n.l(super)
   end
 
-  def fund_and_oda_type
+  def oda_type_summary
+    return I18n.t("is_oda_summary.true") if is_oda.nil?
```

Comment:
Ah, sorry, this must be a misunderstanding of omission - we don't want to show ODA for non-ISPF funds. See for example the Reports index view (e.g https://staging.report-official-development-assistance.service.gov.uk/reports) where only ISPF gets the ODA / non-ODA disambiguation.

The designs for 2974 only showed ISPF because ISPF is the only fund for which we want to make any changes. Sorry if it wasn't clear.

Reply:
Returning "ODA" for cases where `is_oda` is nil makes this the only place in the application where we break the convention that only `is_oda: true` means "ODA". Where do you see this being used in future?

Given that the only place we use this summary is the report view, and we add a guard so we don't show the summary when `is_oda` is nil, won't this add more confusion? Developers will have to remember to always add a guard against nil.

Reply:
As we no longer show the value of `oda_type_summary` for non ISPF reports, this is redundant now.

The logic for `is_oda: nil` means "ODA" is based on my understanding that all funds except ISPF are in fact ODA, so this followed when I thought we had to show it.

That said I think this highlights that the code model is wrong, as for all funds that are not ISPF `is_oda` feels (to me) like it should be true - as that is the case in reality, I appreciate that we are not in a position to resolve that though and of course, I could just be wrong!